diff --git lib/public_key/priv/generate lib/public_key/priv/generate
index fd185bf..69bb226 100755
--- lib/public_key/priv/generate
+++ lib/public_key/priv/generate
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 # Generate ssh moduli files for the sizes in $moduli
 
diff --git make/make_emakefile make/make_emakefile
index 80c4aca..56440d9 100755
--- make/make_emakefile
+++ make/make_emakefile
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 # -*- cperl -*-
 
 use strict;
